java/spacewalk-java.changes.mackdk.issv3-add-registration-notifications

```diff
@@ -0,0 +1,2 @@
+- Added Hub Registration changed notification
+- Improve the notification definition and handling
```

Comment:
I think the second changelog entry could either use some rewording, or alternatively be removed altogether. This changelog is for our customers, and I suspect notification definition and handling without much context might not tell them much. 🙂

Reply:
Indeed, I just removed it